docs/pages/typography.js

```diff
@@ -0,0 +1,1008 @@
+              </Table.Cell>
+              <Table.Cell>
+                <Text>125%</Text>
+              </Table.Cell>
+              <Table.Cell>
+                <Text size="200">Large headlines, usually an H1</Text>
+              </Table.Cell>
+            </Table.Row>
+          </Table.Body>
+        </Table>
+      </MainSection>
+      <MainSection name="Best practices">
+        <Heading size="400" accessibilityLevel={3}>
+          Accessibility
+        </Heading>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="To ensure that all of our customers, including users with low-vision, use a font-size of 14–16px and above for most use cases, especially long-form text. Smaller fonts are ok for less important information, and should be used sparingly."
+            defaultCode={`
+            <Text>We are all here together in freedom, for perhaps the last time! I know, dear; I know that you will always be with me to the end.</Text>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use font sizes below 14px for body copy and UI controls, unless the text is very brief and secondary."
+            defaultCode={`
+            <Text size="100">We are all here together in freedom, for perhaps the last time! I know, dear; I know that you will always be with me to the end.</Text>
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="For SEO and users with screen readers, follow a logical hierarchy by using headings based on a numerical order, not on font size."
+          >
+            <HierarchyDo />
+          </MainSection.Card>
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use font sizes below 14px for body copy and UI controls, unless the text is very brief and secondary."
+          >
+            <HierarchyDont />
+          </MainSection.Card>
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Stick to our [design tokens](/design_tokens) and use color combinations with a 4.5:1 contrast ratio between foreground and background."
+            defaultCode={`
+<Box rounding={2} padding={2} color="infoWeak"><Text>This has enough contrast</Text></Box>
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use text that doesn’t pass 4.5:1 contrast ratio when testing with Figma accessibility plugins."
+            defaultCode={`
+<Box rounding={2} padding={2} color="infoWeak"><Text color="inverse">This does not have enough contrast</Text></Box>
```

Comment:
May need to use an image here due to the a11y tests

Reply:
Yes, see comments below

Reply:
Done!